gapii/fuchsia/gapii_server.cc

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2022 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <lib/async-loop/cpp/loop.h>
+#include <lib/async-loop/default.h>
+#include <lib/fdio/directory.h>
+#include <lib/fdio/io.h>
+#include <lib/vfs/cpp/pseudo_dir.h>
+#include <lib/vfs/cpp/remote_dir.h>
+#include <zircon/processargs.h>
+
+// Serve /pkg as /pkg in the outgoing directory.
```

Comment:
Is this "pkg" arbitrary? if so maybe name it something "AGI-y". Note that in our distribution, there is no "pkg" directory, instead `bazel-bin/pkg` is the root of our distributable and its name is user-controlled.

Reply:
pkg/ is a fuchsia filesystem convention and not related to AGI host packaging.